js/menu.js

```diff
@@ -35,21 +35,11 @@ function init() {
   }
 
   $('.fl-menu-circle-header .nav-toggle').on('click', function() {
-    $('body').toggleClass('disableScroll');
+    $('body').toggleClass('disableScroll circle-menu-active');
     $(this).find('.hamburger').toggleClass('is-active');
     $(this).toggleClass('active');
     $('.fl-menu-circle-nav-holder .nav-circle').toggleClass('active');
     $('.fl-menu-circle-nav-list-holder').toggleClass('active').scrollTop(0);
-
-    if (!$('body').hasClass('hiddenScroll') && $('body').hasScrollBar() && $('body').hasClass('disableScroll') && Fliplet.Env.get('platform') !== 'native') {
```

Comment:
@hcarneiro This logic has been there since the very first commit. Can you explain why this was here? It seems odd that the circular menu toggle is being moved to the left and right when the menu is toggled.

Reply:
@hcarneiro I've reverted this for now. Will investigate and find out if anything needs to be sorted.